### PR TITLE
Fixes #1181 : Handle case when message.text is undefined in Feedback

### LINKED ISSUE #1181: Feedback component throws an error when message text is undefined
**Actual Behaviour**
Feedback component throws an error in console when message text received is undefined and none of the messages are displayed.

**Expected Behaviour**
There should be no console error and the message text can be changed to ""

**Steps to reproduce it**
Try producing a query whose query in undefined. I did by writing an invalid rule in a skill.

**Screenshots of the issue**

screenshot from 2018-05-04 17-18-32

**Would you like to work on the issue?**

Yes, I am working on it

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #1181 

Changes: Handle case when message.text is undefined in Feedback component

Demo Link: https://pr-1182-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 
deepinscreenshot_select-area_20180504175231

### CODE REVIEW

File: src/components/ChatApp/MessageListItem/Feedback.react.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ class Feedback extends React.Component {
 				cursor: 'pointer'
 			}
 
-			let shareMessageSUSI = message.text;
+			let shareMessageSUSI = message.text === undefined ? "" : message.text;
```
Comment (anshumanv) [inline src/components/ChatApp/MessageListItem/Feedback.react.js]: Please switch to single quotes so build passes. :sweat_smile:

Comment (akshatnitd) [reply src/components/ChatApp/MessageListItem/Feedback.react.js]: @anshumanv Made the changes :)

**Comment by akshatnitd** [comment]
@Akshat-Jain Fixed it

**Comment by akshatnitd** [comment]
Also, solves #1190

**Pull request closed without merging**